httpdocs/index.html

```diff
@@ -15,10 +15,17 @@
   <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/default.min.css"/>
   <link rel="stylesheet" href="/css/webots-cloud.css">
 </head>
 <body>
   <div class="scrollable-body" id="scrollable-body"></div>
   <script src="/js/webots-cloud.js" type="module"></script>
 </body>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/highlight.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/1.3.0/showdown.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/1.3.0/showdown.min.js"></script>
```

Comment:
Shall we include both the standard and the min version?

Reply:
Seems it works well with only the `.min.js`